packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -54,7 +56,20 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
   let focus = () => {
     let treeWalker = getFocusableTreeWalker(ref.current);
     if (focusMode === 'child') {
-      let focusable = state.selectionManager.childFocusStrategy === 'last'
+      let focusable;
+      // TODO: This conflicts with the childFocusStrategy stuff set up below, but this behavior made more sense to me
+      // since it respects visual tab order by default.
+      // Perhaps state.selectionManager.childFocusStrategy should default to "undef" in useMultipleSelectionState
+      // so that I can do a childFocusStategy != null check here so it doesn't override what the user sets?
+
+      // If the current active element is a focusable child within the cell, early return.
+      // This means that we've alt/option tab forward/backward into the cell and we should preserve the current
+      // focused child element.
+      if (document.activeElement !== ref.current && ref.current.contains(document.activeElement)) {
+        return;
+      }
```

Comment:
Open question here. Not entirely sure what the desired behavior here is with childFocusStrategy, kinda feel like useMultipleSelectionState shouldn't have a default focus strategy, instead setting it to null similar to how useMenuTriggerState does it. Then I could check a child focus strategy exists and perform the early return if one doesn't, allowing shift+option+tab to follow the visual tab order (currently behaves this way w/o these fixes)